packages/web-components/src/components.d.ts

```diff
@@ -424,21 +420,33 @@ export namespace Components {
          */
         "dateFormat"?: IcDateFormat;
         /**
-          * The text to display as the validation message when `disabledDates` is `from-now` and a disabled date is entered.
+          * The text to display as the validation message when `disableFromNow` is true and a disabled date is entered.
          */
         "dateFromNowMessage"?: string;
         /**
-          * The text to display as the validation message when `disabledDates` is `until-now` and a disabled date is entered.
+          * The text to display as the validation message when `disableUntilNow` is true and a disabled date is entered.
          */
         "dateUntilNowMessage"?: string;
         /**
-          * If `true`, the disabled state will be set.
+          * The days of the week to disable.
```

Comment:
```suggestion
          * If `true`, the days of the week will be set to disable.
```

Reply:
This isn't a boolean value so needs to be worded like this. This prop takes an array of days that the user would like to be disabled, so for example you could disable every monday.